stablehlo/dialect/TypeInference.cpp

```diff
@@ -1991,7 +1999,7 @@ LogicalResult inferDotGeneralOp(
     if (!llvm::is_contained(rhsBatchingDimensions, i) &&
         !llvm::is_contained(rhsContractingDimensions, i))
       dimensions.push_back(rhsShape[i]);
-
+  // dot_general_c12
   inferredReturnShapes.emplace_back(dimensions);
   return success();
 }
```

Comment:
The PR description says "(C14) currently does not have a test and consider removing it Allow distinct precision_config values in DotGeneralOp #1354", but it doesn't look like #1354 is related to C14?

Reply:
This ticket originally tracked it, but was renamed/merged with a different ticket. Updated the link to point to the right one.